integration-tests/maven/src/test/java/io/quarkus/maven/CreateExtensionMojoTest.java

```diff
@@ -45,6 +44,15 @@ static CreateExtensionMojo createMojo(String testProjectName) throws IllegalArgu
         return mojo;
     }
 
+    private static Path getCopyDir(String testProjectName) {
+        while (true) {
+            Path path = Paths.get("target/test-classes/projects/" + testProjectName + "-" + ((int) (Math.random() * 1000)));
```

Comment:
I wonder if using `Random` and `nextInt` would be better than this.

Reply:
It probably would look aesthetically better, but there shouldn't be any real difference.
Would you like me to change it?

Reply:
Not mandatory but it would make things a bit cleaner.

Also, I'm not a big fan of having `while (true)`. Could we maybe limit it to 100 iterations and throw an exception if we can't find one valid path in 100 iterations?

Reply:
Good idea. I'll do that ASAP

Reply:
Fixed